src/main/java/io/github/bbortt/event/planner/domain/Location.java

```diff
@@ -53,7 +54,7 @@
     @JsonIgnoreProperties(value = "locations", allowSetters = true)
     private Responsibility responsibility;
 
-    @OneToMany(mappedBy = "location")
+    @OneToMany(mappedBy = "location", cascade = CascadeType.REMOVE, orphanRemoval = true)
```

Comment:
well, now you're missing `PERSIST, MERGE, REMOVE, REFRESH` and `DETACH`. I don't think we can cover all CRUD functionalities at the moment :D
apart from that, it is bad behavior to have `CascadeType`s on "-to-many" relationships. because Hibernate might execute a ton of queries.
I'd rather look to delete all those in the `LocationService` using something like:
```java
location.getAllSections() // make sure it is eagerly loaded
.forEach(sectionRepository::delete);
```
I think we do have much more control using this attempt.

Reply:
or even more optimised in one query: `sectionRepository.deleteByLocationId(locationId)`.